### PR TITLE
Neo4j/remove verbose syntax

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/TransposedMappingMapper.java [hunk 1]
```diff
@@ -138,31 +121,92 @@ public static List<Mapping> parseEdge(JSONObject edgeMappingsObject) {
...
     if (propertyMappingsObject.has("indexed")) {
       indexed = getFieldAndNameTuples(propertyMappingsObject.get("indexed"));
     }
 
+    for (FieldNameTuple key : keys) {
+      Mapping mapping = new Mapping(fragmentType, RoleType.key, key);
+      addMapping(mappings, mapping);
+    }
+
+    // TODO: remove keys from uniques
```
Comment (fbiville) [inline v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/TransposedMappingMapper.java:198]: will do in a subsequent PR

File: v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/TransposedMappingMapper.java [hunk 2]
```diff
@@ -138,31 +121,92 @@ public static List<Mapping> parseEdge(JSONObject edgeMappingsObject) {
...
     for (FieldNameTuple f : uniques) {
       Mapping mapping = new Mapping(fragmentType, RoleType.property, f);
       addMapping(mappings, mapping);
-      mapping.setIndexed(indexed.contains(f));
+      mapping.setUnique(true);
     }
+
+    // TODO: remove keys and uniques from indexed since they're implicitly indexed
```
Comment (fbiville) [inline v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/TransposedMappingMapper.java:205]: will do in a subsequent PR

**Comment by fbiville** [comment]
@bvolpato feel free to squash this before merging

**Review by bvolpato — APPROVED** [review]

**Review by shakeebshams — APPROVED** [review]
LGTM

**Comment by bvolpato** [comment]
Can you rebase this, please?

**Pull request merged**